### PR TITLE
templates: allow avoiding Google Fonts

### PR DESCRIPTION
**Pull request by vmiklos**
The default behavior is unchanged, but this allows opting out for the
ones who prefer not using 3rd-party services.

### CODE REVIEW

**Comment by vmiklos** [comment]
@gfidente can I help you in some way to get this reviewed? :-)

File: README.md [hunk 1]
```diff
@@ -37,6 +37,8 @@ Supports a number of common global variables but patches are welcomed if you nee
 
 - `HIDE_USER_LOGO` set this to `True` to hide the placeholder without adding your logo
 
+- `AVOID_GOOGLE_FONTS` set this to `True` to avoid using Google Fonts
```
Comment (gfidente) [inline README.md:40]: considering that by doing so generic font families will be used instead (sans-serif, serif, monospace), I wonder if you think it'd make sense to use a non-google-specific name for the variable, maybe USE_GENERIC_FONT_FAMILIES, for it to disable external fonts load?

Comment (vmiklos) [reply README.md]: Sure, why not.

**Comment by vmiklos** [comment]
> I think you're working on this theme more than me these days and I was thinking to grant you write permissions on the repo if that's okay with you

Sure. I would still use PRs, but if I see no comment from you in e.g. a week, then I can go ahead myself. That would be nice. :-)

Not that I have huge plans, the only other change I want to do is to allow integrating with Matomo, which is more or less an open source Google Analytics clone.

**Pull request merged**